src/runtime/graph/graph_runtime.cc

```diff
@@ -103,27 +103,56 @@ class GraphRuntime : public ModuleNode {
   void SetInput(int index, DLTensor* data_in) {
     CHECK_LT(static_cast<size_t>(index), input_nodes_.size());
     uint32_t eid = this->entry_id(input_nodes_[index], 0);
-    TVM_CCALL(TVMArrayCopyFromTo(data_in, &data_entry_[eid], nullptr));
+    data_entry_[eid].CopyFrom(data_in);
   }
   /*!
-   * \brief Copy index-th input to data_out
+   * \brief Get the number of outputs
+   *
+   * \return The number of outputs from graph.
+   */
+  int NumOutputs() {
```

Comment:
```c++
NumOutputs() const
```

Please add to https://docs.tvm.ai/contribute/code_guide.html, support const member  function when possible